src/collection/utils/population.ts

```
@@ -1,35 +1,46 @@
+  RelationPopulationOptions,
+  RelationType,
+} from "../../schema/relations/type-helpers";
+import { Meta } from "../types/expressions";
+import type {
+  Limit,
+  Lookup,
+  PipelineStage,
+  Skip,
+  Sort,
+} from "../types/pipeline-stage";
+import { Projection } from "../types/query-options";
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)